dialect/bin/main/com/powersync/sqlite/PowerSyncDialect.kt

```diff
@@ -0,0 +1,25 @@
+package com.powersync.sqlite
```

Comment:
This was generated after running the Hello World demo app. Not sure why it's not ignored or present in the main branch.

Reply:
Should be `bin` folder perhaps be .gitignored?

Reply:
I think so. I added it now to the ignore list now.